libraries/appbase/application.cpp

```diff
@@ -253,18 +258,31 @@ bool application::initialize_impl(int argc, char** argv, vector<abstract_plugin*
 
       // Copies parameters from config_file into my->_args
       bpo::store(bpo::parse_config_file< char >( config_file_path.make_preferred().string().c_str(),
-                                             my->_cfg_options, true ), my->_args );
+                                                 my->_cfg_options, true ), my->_args );
 
+      return true;
+   }
+   catch (const boost::program_options::error& e)
+   {
+      std::cerr << "Error parsing command line: " << e.what() << "\n";
+      return false;
+   }
+}
 
+bool application::initialize_impl(int argc, char** argv, vector<abstract_plugin*> autostart_plugins)
```

Comment:
where are you calling this function?

Reply:
this is called in application::initialize function. Always has been - i guess its just diff tool problem that it shows as added line cause i did not create this function